### PR TITLE
LPD-43708 Change error messages

### PR DESCRIPTION
**Pull request by larissacribeiro**
Related Issue: [LPD-43708](https://liferay.atlassian.net/browse/LPD-43708)

[LPD-43708]: https://liferay.atlassian.net/browse/LPD-43708?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by larissacribeiro** [comment]
ci:test:sf

**Comment by larissacribeiro** [comment]
ci:test:sf

File: modules/apps/object/object-api/src/main/java/com/liferay/object/exception/ObjectRelationshipEdgeException.java [hunk 1]
```diff
@@ -7,32 +7,37 @@
...
 import com.liferay.portal.kernel.exception.PortalException;
 
+import java.util.List;
+
 /**
  * @author Marco Leo
  */
 public class ObjectRelationshipEdgeException extends PortalException {
 
-	public ObjectRelationshipEdgeException() {
```
Comment (guilhermedcamacho) [inline modules/apps/object/object-api/src/main/java/com/liferay/object/exception/ObjectRelationshipEdgeException.java:15]: Hey @larissacribeiro, I think the pattern we follow is to keep the constructors generated by buildService. Can you check other examples and if I'm correct, fix this?

**Review by guilhermedcamacho — CHANGES_REQUESTED** [review]

**Comment by larissacribeiro** [comment]
ci:test:object

**Comment by larissacribeiro** [comment]
ci:test:relevant

File: modules/apps/object/object-api/src/main/java/com/liferay/object/exception/ObjectRelationshipEdgeException.java (hunk 1 above)
Comment (larissacribeiro) [reply modules/apps/object/object-api/src/main/java/com/liferay/object/exception/ObjectRelationshipEdgeException.java:15]: We discussed this in person with other devs and aligned that it was okay to remove the unused ones, it has already been done other in other files. thanks!

**Comment by larissacribeiro** [comment]
more conflicts on the language key files, I'll handle it now 😮‍💨

**Comment by larissacribeiro** [comment]
ci:test:sf

**Comment by larissacribeiro** [comment]
ci:test:relevant

**Comment by larissacribeiro** [comment]
ci:test:object

**Comment by larissacribeiro** [comment]
ci:test:sf

**Comment by larissacribeiro** [comment]
ci:test:relevant

**Comment by larissacribeiro** [comment]
ci:forward:force

**Pull request closed without merging**